src/patternfly/components/Page/page.scss

```diff
@@ -438,6 +438,8 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
 .pf-c-page__main-breadcrumb,
 .pf-c-page__main-section,
 .pf-c-page__main-wizard {
+  flex-shrink: 0;
```

Comment:
trying to follow along here so this may be incorrect, but from your codepen you used `flex-basis: auto;` , i dont see that anywhere here do you need that too ?

Reply:
Ah yeah, `flex-basis` on those elements is already `auto`, so it's not necessary to set it specifically. The elements just need `flex-shrink: 0` with `flex-basis: auto`